### PR TITLE
Bring back support for Mac darwin-x64

### PR DESCRIPTION
**Pull request by WilsonZiweiWang**
Ticket: 14204

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -824,7 +824,7 @@
     "test:grammar": "vscode-tmgrammar-test ./test/grammars/test-cases/*.bb",
     "snap-grammar": "vscode-tmgrammar-snap ./test/grammars/snaps/*.bb -u",
     "vscode:prepublish": "npm install --omit=dev && cd ../server && npm install --omit=dev && npm run installServer",
-    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web"
+    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web && vsce package --target darwin-x64"
```
Comment (deribaucourt) [inline client/package.json:827]: Also add darwin-arm64. It should work out of the box since nothing is compiled.

Comment (deribaucourt) [reply client/package.json:827]: We could also add alpine-x64, alpine-arm64. There's no reason they wouldn't work.

**Review by deribaucourt — CHANGES_REQUESTED** [review]

**Review by deribaucourt — APPROVED** [review]

File: client/package.json [hunk 2]
```diff
@@ -824,7 +824,7 @@
     "test:grammar": "vscode-tmgrammar-test ./test/grammars/test-cases/*.bb",
     "snap-grammar": "vscode-tmgrammar-snap ./test/grammars/snaps/*.bb -u",
     "vscode:prepublish": "npm install --omit=dev && cd ../server && npm install --omit=dev && npm run installServer",
-    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web"
+    "package": "npm run vscode:prepublish && vsce package --target linux-x64 && vsce package --target linux-arm64 && vsce package --target linux-armhf && vsce package --target win32-x64 && vsce package --target win32-arm64 && vsce package --target web && vsce package --target darwin-x64 && vsce package --target darwin-arm64 && vsce package --target alpine-x64 && vsce package --target alpine-arm64"
```
Comment (deribaucourt) [inline client/package.json:827]: Actually let's just get back to universal now? the contents are the same. Do we need a specific bundle for web? Probably not.

Comment (WilsonZiweiWang) [reply client/package.json:827]: No, the vscode only needs to know the value in `browser` property. We could revert it back to the package for all. But what about the platforms that are not here? Such as `win32-ia32`. Though, I think the safer way is to put the ones that we know it works.

Comment (deribaucourt) [reply client/package.json:827]: At this point I think we can consider everythin to work.

Comment (deribaucourt) [reply client/package.json:827]: I suggest going back to a universal package. Make sure it works on web if possible. Update the packaging and deployment script if required.

**Pull request merged**